samples/OpenIdConnect.AzureAdSample/Startup.cs

```diff
@@ -70,7 +70,7 @@ public void ConfigureServices(IServiceCollection services)
                         var authContext = new AuthenticationContext(Authority, AuthPropertiesTokenCache.ForCodeRedemption(context.Properties));
 
                         var result = await authContext.AcquireTokenByAuthorizationCodeAsync(
-                            context.ProtocolMessage.Code, new Uri(currentUri), credential, Resource);
```

Comment:
I understand why this was changed, but ProtocolMessage needs to stay. Alias it if you want to, but ProtocolMessage is the thing in the event that gets used and will break the most consumers. Changing the context names won't affect most people when they just use the lambadas.